### ISSUE TITLE
Need to integration test Akka.DistributedData w/ Durability and Akka.Cluster.Sharding

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/akka.net#4873]
Version: 1.4.18

We're getting some bug reports that `remember-entities` doesn't work with Akka.DistributedData when an Akka.NET cluster undergoes a full reboot - naturally, that makes sense because all of the in-memory CRDTs are dumped when the entire cluster is shut down and spun back up again.

However, we've _also_ received reports that when enabling "durable" DData, which should be using LightningDb to save CRDT data to disk, this still doesn't work.

So we need to create a reproduction spec in the MNTR test suite to verify this:

1. Create a 3-node sharded cluster using DData w/ remember-entities=on and durable DData enabled;
2. Create at least 1 shard per node;
3. Shut down the entire cluster;
4. Reboot the cluster;
5. Validate that all entities are started back up.

If that test can pass, we need to document how to enable these settings inside the Akka.Cluster.Sharding documentation and also close out https://github.com/akkadotnet/akka.net/issues/4810 while we're at it - if it can't pass, we need to fix it.

**Comment by Aaronontheweb** [comment]
Relevant Gitter thread: https://gitter.im/akkadotnet/akka.net?at=6059d56a9ebdfd1640921ba8

**Comment by Arkatufus** [comment]
We actually turned this off...
https://github.com/akkadotnet/akka.net/issues/4262

**Comment by Arkatufus** [comment]
I just compared our code and the JVM code, it turns out that we're missing a large chunk of DData sharding code, it might take awhile to port those into our code base.

**Comment by Aaronontheweb** [comment]
@Arkatufus I think we turned it off at the time simply because we didn't have Durable DData support completed yet when we released it. That was fixed later but we never went back and revisited it.

**Comment by Arkatufus** [comment]
Yes, it was the `Akka.DistributedData.LightningDB` port that I did, but it turns out, we're missing more code than just that.

**Comment by Arkatufus** [comment]
These are the files that needs to be ported:
![image](https://user-images.githubusercontent.com/8293237/112875795-d8de5e00-90ee-11eb-958b-f21f95b7a8dd.png)

**Comment by Aaronontheweb** [comment]
@Arkatufus go ahead and get started on it

**Comment by Aaronontheweb** [comment]
```
I've got the data.mdb for the shard updating now, so I can see data is saved, I see some pruning activity after the cluster is up, but no entities are started. i do see this Message [Register] from akka://test/system/sharding/Test to akka://test/system/sharding/TestCoordinator/singleton/coordinator was not delivered. but that seems to belong to PersistentShardCoordinator, whereas I'd not expect that id DDataShardCoordinator is used.

```

From the Gitter thread I referenced earlier.

**Comment by zbynek001** [comment]
all of that is already part of https://github.com/akkadotnet/akka.net/pull/4629

**Comment by Aaronontheweb** [comment]
@zbynek001 good to know!

 @Arkatufus I'll revisit reviewing that PR - I know you had some issues with it, but maybe it can be fixed up. Sound good?

**Comment by Arkatufus** [comment]
@Aaronontheweb Sounds good to me, I'm just a bit too strict in reviewing codes, would be glad to have a second pair of eyes to look it over.

**Comment by Aaronontheweb** [comment]
Bumping to v1.4.20 based on feasibility feedback from @Arkatufus
